VERSION ""


NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_: 
BU_: Dashboard PEI TelemNode 
BO_ 1894 Dashboard_Vehicle_State: 7 Dashboard
   SG_ HV_Requested : 7|8@0+ (1,0) [0|1] "" Vector__XXX
   SG_ Throttle1_Level : 15|8@0+ (1,0) [0|100] "%" Vector__XXX
   SG_ Throttle2_Level : 23|8@0+ (1,0) [0|100] "%" Vector__XXX
   SG_ Brake_Level : 31|8@0+ (1,0) [0|100] "%" Vector__XXX
   SG_ VCU_ticks : 47|16@0+ (1,0) [0|65535] "ms" Vector__XXX
   SG_ State : 39|8@0+ (1,0) [0|255] "" Vector__XXX

BO_ 896 PEI_BMS_Status: 5 PEI
   SG_ BMS_Status : 7|8@0+ (1,0) [0|255] "" Vector__XXX
   SG_ SPI_Error_Flags : 15|16@0+ (1,0) [0|65535] "" Vector__XXX
   SG_ Max_Faulting_IC_Address : 31|8@0+ (1,0) [0|9] "" Vector__XXX
   SG_ Communication_Break_ID : 39|8@0- (1,0) [-1|9] "" Vector__XXX

BO_ 903 PEI_Status: 5 PEI
   SG_ Current_ADC : 15|16@0+ (1,0) [0|65535] "A" Vector__XXX
   SG_ IMD_OK : 5|1@0+ (1,0) [0|1] "" Vector__XXX
   SG_ BMS_OK : 4|1@0+ (1,0) [0|1] "" Vector__XXX
   SG_ SHUTDOWN_FINAL : 3|1@0+ (1,0) [0|1] "" Vector__XXX
   SG_ AIR_NEG : 2|1@0+ (1,0) [0|1] "" Vector__XXX
   SG_ AIR_POS : 1|1@0+ (1,0) [0|1] "" Vector__XXX
   SG_ PRECHARGE : 0|1@0+ (1,0) [0|1] "" Vector__XXX
   SG_ Current_Reference : 31|16@0+ (1,0) [0|65535] "" Vector__XXX

BO_ 1024 TelemNode_Cooling_Loop_Temps: 8 TelemNode
   SG_ Inlet_Water_Temp : 7|16@0- (0.1,0) [-3276.8|3276.7] "C" Vector__XXX
   SG_ Outlet_Water_Temp : 23|16@0- (0.1,0) [-3276.8|3276.7] "C" Vector__XXX
   SG_ Air_Into_Radiator_Temp : 39|16@0- (0.1,0) [-3276.8|3276.7] "C" Vector__XXX
   SG_ Air_Out_Of_Radiator_Temp : 55|16@0- (0.1,0) [-3276.8|3276.7] "" Vector__XXX

BO_ 1026 TelemNode_Cool_Loop_Pressures: 4 TelemNode
   SG_ Inlet_Water_Pressure : 7|16@0+ (0.01,0) [0|1] "PSI" Vector__XXX
   SG_ Outlet_Water_Pressure : 23|16@0+ (0.01,0) [0|1] "PSI" Vector__XXX

BO_ 1025 TelemNode_Wheel_Speed_Rear: 4 TelemNode
   SG_ Wheel_Speed_Rear_Right : 7|16@0+ (1,0) [0|65535] "CPS/RPM" Vector__XXX
   SG_ Wheel_Speed_Rear_Left : 23|16@0+ (1,0) [0|65535] "CPS/RPM" Vector__XXX

BO_ 192 Dashboard_Torque_Request: 8 Dashboard
   SG_ Torque : 0|16@1+ (0.1,0) [0|6553.5] "Nm" Vector__XXX
   SG_ Speed : 16|16@1+ (1,0) [0|65535] "RPM" Vector__XXX
   SG_ Direction : 32|8@1+ (1,0) [0|1] "" Vector__XXX
   SG_ Torque_Limit : 48|16@1+ (0.1,0) [0|6553.5] "Nm" Vector__XXX
   SG_ Inverter_Enable : 40|1@1+ (1,0) [0|1] "" Vector__XXX
   SG_ Discharge_Enable : 41|1@1+ (1,0) [0|1] "" Vector__XXX
   SG_ Speed_Mode_Enable : 42|1@1+ (1,0) [0|1] "" Vector__XXX

BO_ 1280 Dashboard_Random_Shit: 6 Dashboard
   SG_ Front_Strain_Gauge : 7|16@0+ (1,0) [0|65535] "" Vector__XXX
   SG_ Front_Wheel_Speed : 23|16@0+ (1,0) [0|65535] "RPM" Vector__XXX
   SG_ TC_Torque_Request : 39|16@0+ (0.1,0) [0|6553.5] "Nm" Vector__XXX

BO_ 897 PEI_Diagnostic_BMS_Data: 4 PEI
   SG_ HI_Temp : 7|8@0+ (1,0) [0|255] "C" Vector__XXX
   SG_ SOC : 15|8@0+ (1,0) [0|100] "%" Vector__XXX
   SG_ Pack_Voltage : 23|16@0- (1,0) [-32768|32767] "V" Vector__XXX

BO_ 1027 TelemNode_Strain_Gauges_Rear: 8 TelemNode
   SG_ RL_Toe_Strain_Gauge : 7|16@0+ (1,0) [0|65535] "" Vector__XXX
   SG_ RLUF_A_Arm_Strain_Gauge : 23|16@0+ (1,0) [0|65535] "" Vector__XXX
   SG_ RLUB_A_Arm_Strain_Gauge : 39|16@0+ (1,0) [0|65535] "" Vector__XXX
   SG_ RLLF_A_Arm_Strain_Gauge : 55|16@0+ (1,0) [0|65535] "" Vector__XXX

BA_DEF_ BO_ "GenMsgBackgroundColor" STRING ;
BA_DEF_ BO_ "GenMsgForegroundColor" STRING ;
BA_DEF_ BO_ "matchingcriteria" INT 0 0;
BA_DEF_ BO_ "filterlabeling" INT 0 0;
BA_DEF_ BO_ "matchingcriteria" INT 0 0;
BA_DEF_ BO_ "filterlabeling" INT 0 0;
BA_DEF_ BO_ "matchingcriteria" INT 0 0;
BA_DEF_ BO_ "filterlabeling" INT 0 0;
BA_ "GenMsgForegroundColor" BO_ 1894 "#000000";
BA_ "GenMsgForegroundColor" BO_ 896 "#000000";
BA_ "GenMsgForegroundColor" BO_ 903 "#000000";
BA_ "GenMsgForegroundColor" BO_ 1024 "#000000";
BA_ "GenMsgForegroundColor" BO_ 1026 "#000000";
BA_DEF_DEF_ "GenMsgBackgroundColor" "#ffffff";
BA_DEF_DEF_ "GenMsgForegroundColor" "#000000";
BA_DEF_DEF_ "matchingcriteria" 0;
BA_DEF_DEF_ "filterlabeling" 1;
BA_DEF_DEF_ "matchingcriteria" 0;
BA_DEF_DEF_ "filterlabeling" 1;
